app/main.py

```diff
@@ -1,3 +1,65 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union[int, "Distance"]) -> "Distance":
```

Comment:
```suggestion
    def __add__(self, other: Union[int, Distance]) -> Distance:
```

Reply:
Change everywhere, add `from __future__ import annotations ` in the file beginning